src/lib/api-client.js

```diff
@@ -165,3 +165,10 @@ export function longTermCheckout(studentId, itemAddresses, adminCode, longtermDu
         longtermProfessor
     });
 }
+
+export function patchItemDueDate(studentId, address, date){
+    return patch('student/item', {studentId}, {
```

Comment:
~~There's no coverage for this line; got a test for that?~~ You _do_ stub it; maybe a functional test will cover it?